article/sections/_1_intro_generic.tex

```diff
@@ -0,0 +1,48 @@
+\section{Introduction}
+\label{sec:intro}
+
+In transportation, we often build models that are used for policy evaluation.
+Specifically, we often use behavioral models to evaluate the impact of external interventions on travel outcomes. 
+These evaluations are excplicitly causal: we are interested in how the 
+system reacts to \textit{interventions}. 
+Yet, when these models are 
+developed, causality is often not considered. Furthermore, when 
+causal concepts are accounted for, the process is done implicitly without a 
+formal framework. 
+
+This chapter is concerned with filling this gap. 
+Specifically, we focus on addressing the existing disconnect between the fields of travel demand modeling and causal inference. 
+The chapter is motivated by the current lack of use of methods and findings from 
+the causal inference literature in travel demand modeling. 
```

Comment:
I feel these three sentences are saying pretty much the same thing. I am yet to find a good replacement that could combine the three of them.